logging/logging.go

```diff
@@ -43,6 +43,11 @@ type LoggerSetter interface {
 
 type loggerKey struct{}
 
+// IsDebugLevel return true if logger configured to log debug messages
+func IsDebugLevel(logger *log.Entry) bool {
```

Comment:
An idea of (possibly) more convenient interface for debug logging:

```go
some.logger.OnlyForLevel(log.DebugLevel).
	WithField("machine_readable", data).
	Debugf("human-readable message")
```

No idea how to implement this tho (at the moment), but with this we should avoid one unnecessary nesting level of the `if` check before logging.

Though, I believe the current approach has its merits as well. We may use `IsDebugLevel` to avoid doing some heavy computations for debug logging if it is not actually enabled.

Reply:
now this method used to check that logging configured with DebugLevel and we can add to logger's context more data (now used only for mode of decryption whole|inline) which will be used in inner functions but not unnecessary for default InfoLevel